lms/templates/ccx/schedule.html

```diff
@@ -147,22 +147,22 @@ <h3 class="hd hd-3" id="ccx_schedule_unit">${_('Schedule a Unit')}</h3>
     });
     view.render();
     //ccx_schedule.render();
-    $('.datepair .time').timepicker({
-        'showDuration': true,
-        'timeFormat': 'G:i',
-        'autoclose': true
-    });
-    $('.datepair .date').datepicker({
-        'dateFormat': 'yy-mm-dd',
-        'autoclose': true
-    });
-    $('.datepair .date').change(function() {
-       var date = $(this).datepicker( "getDate" );
-       if (date) {
-           $(this).val(date.getFullYear() + "-" +
-                   ('0' + (date.getMonth() + 1)).slice(-2) + "-" +
-                   ('0' + date.getDate()).slice(-2));
-       }
-    });
+    // $('.datepair .time').timepicker({
```

Comment:
In case we have to revert if need be, or if it affects any other part of platform if it uses given template. Compiler will ignore comments anyway.